src/Compilers/CSharp/Portable/Compiler/DocumentationCommentCompiler.cs

```diff
@@ -413,13 +411,6 @@ public override void DefaultVisit(Symbol symbol)
 
                 bool reportDiagnosticsForCurrentTrivia = trivia.SyntaxTree.ReportDocumentationCommentDiagnostics();
```

Comment:
is this variable needed anymore?

Reply:
Yes, it is used later.

---
In reply to: [184566009](https://github.com/dotnet/roslyn/pull/26425#discussion_r184566009) [](ancestors = 184566009)